### PR TITLE
Upgrade Baseline and Error Prone

### PR DESCRIPTION
**Pull request by pkoenig10**
See https://github.com/palantir/gradle-baseline/pull/2839

### CODE REVIEW

File: .baseline/copyright/999_palantir.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+(c) Copyright ${today.year} Palantir Technologies Inc. All rights reserved.
```
Comment (carterkozak) [inline .baseline/copyright/999_palantir.txt]: This should not be added

Comment (pkoenig10) [reply .baseline/copyright/999_palantir.txt]: Hmm this was generated by `baselineUpdateConfig`, will figure out why.

Comment (pkoenig10) [reply .baseline/copyright/999_palantir.txt]: This is also generated if you run `baselineUpdateConfig` on develop. I can remove it here, but I suspect that an excavator will automatically generate it in the future.

Comment (carterkozak) [reply .baseline/copyright/999_palantir.txt]: The excavator deletes new copyright files added by the task, when running the update task we need to do that step manually

**Review by carterkozak — APPROVED** [review]

**Pull request merged**